src/actors/symbolication.rs

```diff
@@ -371,6 +397,14 @@ impl SymbolicationActor {
                             stacktrace.is_requesting = requesting_thread_index.map(|r| r == i);
                         }
 
+                        for (mut module, unwind_status) in response
+                            .modules
+                            .iter_mut()
+                            .zip(unwind_statuses_by_module_index)
+                        {
+                            module.unwind_status = Some(unwind_status);
```

Comment:
Suggestion: How about we change `SymbolicateStacktraces` to contain the full object info already, and then `do_process_minidump` already writes unwind status in there. It seems a bit clumsy that this is only done after the fact.

Reply:
Done